apps/institution-service/src/controllers/institutionController.test.ts

```diff
@@ -865,4 +867,72 @@ describe("institutionController", () => {
       });
     });
   });
+
+  describe("deleteInstitution", () => {
+    it("removes the institution and responds with a 204 on success", async () => {
+      const newInstitutionId = uuidv4() as UUID;
+
+      const institutionBody = {
+        ...testInstitution,
+        id: newInstitutionId,
+        name: `deleteTest-${newInstitutionId}`,
+      };
+
+      await Institution.create(institutionBody);
+
+      expect(await Institution.findByPk(newInstitutionId)).not.toBe(null);
+
```

Comment:
It's to ensure that something is actually removed. If I only check on 897 that it doesn't exist, then it may never have existed. Checking that it exists here gives me more confidence that it being gone later isn't a coincidence.